rts/Sim/Misc/LosHandler.cpp

```diff
@@ -824,10 +829,23 @@ void CLosHandler::Update()
 
 void CLosHandler::UpdateHeightMapSynced(SRectangle rect)
 {
-	for (ILosType* lt: losTypes) {
-		ZoneScopedN("LosHandler::UpdateHeightMapSynced");
-		lt->UpdateHeightMapSynced(rect);
+	int losHandlerUpdateHeightMapSyncedMT = configHandler->GetInt("LosHandlerUpdateHeightMapSyncedMT");
```

Comment:
Is this called more than a few times per second? If so, consider hot loading confighandler. 

See `COverheadController::ConfigUpdate` and `COverheadController::ConfigNotify` for reference.